DeusEx/Classes/DXRWeapons.uc

```diff
@@ -0,0 +1,32 @@
+class DXRWeapons extends DXRBase;
+
+function AnyEntry()
+{
+    local DeusExWeapon w;
+    Super.AnyEntry();
+
+    foreach AllActors(class'DeusExWeapon', w) {
+        RandoWeapon(w);
+    }
+}
+
+function RandoWeapon(DeusExWeapon w)
+{
+    local float f;
+    //l("RandoWeapon("$w$")");
+    dxr.SetSeed( dxr.Crc(dxr.seed $ "RandoWeapon " $ w.class.name ) );
+
+    f = float(w.default.HitDamage) * (rngf()+0.5);
+    w.HitDamage = int(f);
+    //w.HitDamage = 90001;
+    f = w.default.ShotTime * (rngf()+0.5);
+    w.ShotTime = f;
+    /*f = w.default.ReloadTime * (rngf()+0.5);

```

Comment:
these would be weird to keep track of and show in the description, because weapon mods overwrite these values

like WeaponModReload

```
function ApplyMod(DeusExWeapon weapon)
{
	if (weapon != None)
	{
		weapon.ReloadTime    += (weapon.Default.ReloadTime * WeaponModifier);
		if (weapon.ReloadTime < 0.0)
			weapon.ReloadTime = 0.0;
		weapon.ModReloadTime += WeaponModifier;
	}
}
```